File: c/1044_multiples.c
/*  
    @autor: Guiventura6;
    @data: 21/01/2022;
    @Problema: Multiples;
*/
#include <stdio.h>

int main(){
    
    int a, b;

    scanf("%d%d", &a, &b);

    if (a % b == 0 || b % a == 0) {
        printf("Sao Multiplos\n");
    } else {
        printf("Nao sao Multiplos\n");
    }
        
    return 0;
}

File: python/1044_multiples.py
'''
    Autor: Guiventura6
    Date: 04/07/2023
    Modified: ()
    Problem: Multiples
    Resume: Program that read two interger values and verify if is multiples
'''


def is_multiple(a, b):
    if a % b == 0 or b % a == 0:
        print("Sao Multiplos")
    else:
        print("Nao sao Multiplos")


a, b = map(int, input().split())
is_multiple(a, b)
